homeassistant/components/scrape/sensor.py

```diff
@@ -90,7 +91,7 @@ async def async_setup_platform(
         hass.config_entries.flow.async_init(
             DOMAIN,
             context={"source": SOURCE_IMPORT},
-            data=config,
+            data={k: v for k, v in config.items() if k != CONF_SCAN_INTERVAL},
```

Comment:
Shouldn't we convert the scan interval back to seconds instead? 
```suggestion
            data={
                k: v.total_seconds() if k == CONF_SCAN_INTERVAL else v
                for k, v in config.items()
            },
```

Reply:
nevermind, that's a generic platform config option and not part of the integration.